core/oidc.go

```diff
@@ -464,10 +466,10 @@ func (o *OIDC) token(ctx context.Context, req *tokenRequest, handler func(req *T
 		GrantType:          req.GrantType,
 		SessionRefreshable: strsContains(sess.Authorization.Scopes, "offline_access"),
 		IsRefresh:          isRefresh,
-
-		authTime: sess.Authorization.AuthorizedAt,
-		authReq:  sess.Request,
-		now:      o.now,
+		Nonce:              sess.Request.Nonce,
```

Comment:
I think the extra newline between exported and unexported fields was intentional. Could you add it back?